components/octopost/Works/Works.tsx

```diff
@@ -0,0 +1,91 @@
+'use client'
+
+import { useState, useRef, useEffect } from 'react';
+import Bubble from "./assets/Bubble";
+import VectorLeft from "./assets/VectorLeft";
+import VectorRight from "./assets/VectorRight";
+import Card from "./Card";
+
+import { CARDS } from './utils/Cards';
+
+function Works() {
+  const [activeSlide, setActiveSlide] = useState<number>(0);
+  const scrollContainerRef = useRef<HTMLUListElement>(null);
+
+  const handleScrollEvent = (): void => {
+    if (scrollContainerRef.current) {
+      const scrollLeft = scrollContainerRef.current.scrollLeft;
+      const width = scrollContainerRef.current.clientWidth;
+      const newActiveSlide = Math.round(scrollLeft / width);
+      setActiveSlide(newActiveSlide);
+    }
+  };
+
+  return (
+    <section className="h-screen bg-octopost-primaryWhite flex items-center relative overflow-hidden">
+      <div className="m-auto flex flex-col max-w-7xl z-20 overflow-hidden gap-9 desktop:gap-32">

```

Comment:
```suggestion
      <div className=" desktop:gap-32">
     <div className=" xl:gap-32">
```
O tailwind tem as media queries já predefinida deles, essas nao servem?
https://tailwindcss.com/docs/responsive-design

screens: {
        'mobile': {'min': '320px', 'max': '480px'},
        'tablet': {'min': '768px', 'max': '1279px'},
        'desktop': {'min': '1280px'},
      },